test/test_add.mjs

```diff
@@ -80,7 +78,8 @@ export function testNodeAdd (localBuildPath
       expect(nodeCmd.getUnusedConfigs(NodeCommand.ADD_CONFIGS_NAME)).toEqual([
         flags.app.constName,
         flags.chainId.constName,
-        flags.devMode.constName
+        flags.devMode.constName,
+        flags.adminKey.constName
```

Comment:
nitpick: I prefer to sort these lists alphabetically, makes it easier to maintain

Reply:
The assertion requires them to be in this exact order, otherwize it fails

Reply:
oh, yes, that is because adminKey is an extra and not a flag. I didn't realize that.